packages/open-next/src/build.ts

```diff
@@ -198,47 +167,48 @@ function printHeader(header: string) {
   );
 }
 
-function printNextjsVersion() {
-  const { nextVersion } = options;
-  logger.info(`Next.js version : ${nextVersion}`);
+function printNextjsVersion(options: BuildOptions) {
+  logger.info(`Next.js version : ${options.nextVersion}`);
 }
 
-function printOpenNextVersion() {
-  const { openNextVersion } = options;
-  logger.info(`OpenNext v${openNextVersion}`);
+function printOpenNextVersion(options: BuildOptions) {
+  logger.info(`OpenNext v${options.openNextVersion}`);
 }
 
-function initOutputDir(tempDir: string) {
+function initOutputDir(srcTempDir: string, options: BuildOptions) {
```

Comment:
I guess we could move `srcTempDir` inside options as well

Reply:
I'll address in a follow up PR